settings.gradle

```diff
@@ -12,3 +14,15 @@ include 'dataline-server'
 include 'dataline-webapp'
 include 'dataline-workers'
 include 'dataline-tests'
+
+// include all integration subprojects
+def prefix = rootProject.projectDir.absolutePath
+def dir = new File(prefix + "/dataline-integrations")
+dir.eachFileRecurse(FileType.FILES) { file ->
+  if (file.absolutePath.endsWith("build.gradle") && file.length() != 0) {
```

Comment:
can you do `file.isEmpty` instead? (not sure if it exist)

Reply:
It sadly doesn't exist.